node.gyp

```diff
@@ -996,45 +982,6 @@
            'dependencies': ['node_aix_shared'],
          }, {
            'dependencies': ['<(node_lib_target_name)'],
-         }],
-        ['node_intermediate_lib_type=="static_library" and node_shared=="false"', {
-          'xcode_settings': {
-            'OTHER_LDFLAGS': [
-              '-Wl,-force_load,<(PRODUCT_DIR)/<(STATIC_LIB_PREFIX)'
-              '<(node_core_target_name)<(STATIC_LIB_SUFFIX)',
-            ],
-          },
-          'msvs_settings': {
-            'VCLinkerTool': {
-              'AdditionalOptions': [
-                '/WHOLEARCHIVE:lib<(node_core_target_name)<(STATIC_LIB_SUFFIX)',
-              ],
-            },
-          },
-          'conditions': [
-            ['OS!="aix"', {
-              'ldflags': [
-                '-Wl,--whole-archive,'
-                '<(obj_dir)/<(STATIC_LIB_PREFIX)<(node_core_target_name)<(STATIC_LIB_SUFFIX)',
-                '-Wl,--no-whole-archive',
-              ],
-            }],
-            ['OS=="win"', {
-              'sources': ['src/res/node.rc'],
-              'conditions': [
-                ['node_use_etw=="true"', {
```

Comment:
Um, doesn't this break etw builds? (I am not sure who actually uses this, though, and it's been broken quite often, but it's listed in https://github.com/nodejs/node/blob/master/doc/guides/diagnostic-tooling-support-tiers.md)

Reply:
Code just moved to https://github.com/nodejs/node/blob/380d1a6b9ce4436635e02a65d32198699b0fdb07/node.gyp#L627